### DISCUSSION TITLE
DAT Linux package sources

### DISCUSSION THREAD

**Discussion by playmobilmeister** [dat-linux/community#61] [App, Extra, Book suggestions]
I'm interested in trying DAT. Trying to understand the package sources, as a current Lubuntu 23.10 (a non -LTS version) and Manjaro (The XFCE version) user. Taking aside the question of the default installed apps -  what about versions? 

example: QGIS in Ubuntu (Mantic Minotaur, the current as for today) is 3.28 while upstrean in QGIS's website they already have the next LTS, version 3.34 and one beyond that. What's your update policy and the current version?

**Comment by playmobilmeister** [comment]
Additional example is Rstudio, do you keep up with Posit's releases or wait ? it looks [quite behind](https://repology.org/project/rstudio/versions) in most repositories

**Comment by dat-linux** [comment]
Hi,

Policy is to have the latest stable versions as far as possible. QGIS 3.36 is current.

**Comment by dat-linux** [comment]
Apps are pulled from the best source, could be direct download, AppImage, ubuntu package repo, custom package repo, or pip install.

**Comment by dat-linux** [comment]
As I said, best possible latest version.

RStudio is currently pulling 2023.12.1Build 402.
